hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -32,7 +32,8 @@
                 "{proj_root}/hw/dv/tools/dvsim/common_sim_cfg.hjson",
                 // Common CIP test lists
                 "{proj_root}/hw/dv/tools/dvsim/tests/csr_tests.hjson",
-                "{proj_root}/hw/dv/tools/dvsim/tests/mem_tests.hjson",
+                // TODO, comment this out because spi host memory is dummy #5484
```

Comment:
nit: prefer the `// TODO #xxxx: Comment` format. Makes it easier to grep.

Reply:
sure, updated, thanks